src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js

```diff
@@ -1,15 +1,113 @@
 // @flow
 import * as React from 'react';
+import getProp from 'lodash/get';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+
+import Comment from '../comment';
+import ActivityItem from './ActivityItem';
+import PlainButton from '../../../../components/plain-button';
+
+import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../common/flowTypes';
+import type { Translations } from '../../flowTypes';
+import type { SelectorItems, User } from '../../../../common/types/core';
+import type { Comment as CommentType } from '../../../../common/types/feed';
+
+import messages from './messages';
+import './ActivityThread.scss';
 
 type Props = {
     children: React.Node,
+    currentUser?: User,
+    getAvatarUrl: GetAvatarUrlCallback,
+    getMentionWithQuery?: Function,
+    getUserProfileUrl?: GetProfileUrlCallback,
     hasReplies: boolean,
+    id: string,
+    mentionSelectorContacts?: SelectorItems<>,
+    onGetReplies?: (id: string) => void,
+    onReplyDelete?: Function,
+    onReplyEdit?: Function,
+    replies?: Array<CommentType>,
+    total_reply_count?: number,
```

Comment:
`replies` and `total_replies_count` are properties on Item passed from ActiveState component. 
onGetReplies will be be passed down from ActivitySidebar - it will be api method separate for annotations and comments to fetch all replies connected to entity.